.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -1,26 +1,34 @@
+  - [ ] Once you are added to the GSA-TTS org, ask the person(s) with the "Maintainer" role to add you to [the FAC-team in GitHub](https://github.com/orgs/GSA-TTS/teams/fac-team/members). This will give you read/write access to our repositories.
+- [ ] (designers) Review the [design onboarding document](https://docs.google.com/document/d/1EILl0nZr59T4PFJJMtFbmnQDJPksgzIFPuoFDN0bk0g/edit#heading=h.bhu3dgydlbvr)
+- [ ] (engineers) Familiarize yourself with Python, Django, and Cloud.gov—all tools used in this project.
+  - [ ] If you need to catch up on the latest in Python development, check out the [Python developer's guide](https://devguide.python.org/).
+  - [ ] Work through the [Django Tutorial](https://docs.djangoproject.com/en/4.0/intro/tutorial01/) and writing your first Django app.
+  - [ ] If you're not already, get [setup with Cloud.gov](https://cloud.gov/docs/getting-started/setup/)
+  - [ ] Practice deploying a [python application](https://github.com/cloud-gov/cf-hello-worlds/tree/main/python-flask) to Cloud.gov using the Cloud.gov command line interface (CLI): https://cloud.gov/docs/getting-started/your-first-deploy/.
+  - [ ] Survey existing TTS projects which use Django+Cloud.gov like: [Tock](https://github.com/18F/tock) and the [DOJ CRT Portal](https://github.com/usdoj-crt/crt-portal).
+- [ ] (engineers) Get set up for [local development](https://github.com/GSA-TTS/FAC/blob/main/docs/development.md#local-development) so you can start contributing
 
 {ONBOARDING BUDDY'S GH USERNAME} will be your onboarding buddy and will tackle the list below!
 
 **For your onboarding buddy**
-- [ ] Add to the FAC team folder in drive (request of Dan R. or Maggie S. in 10x FO)
-- [ ] Add to [FAC-team@GSA.gov](https://groups.google.com/a/gsa.gov/g/fac-team) google group
-- [ ] <Engineering> Added to `gsa-10x-prototyping` Organization in cloud.gov
-- [ ] Add to the [FAC team](https://github.com/orgs/GSA-TTS/teams/fac-team) in github
-- [ ] Add to standing meeting invites (should happen automatically via GGroup?)
+- [ ] Add to [FAC-team@GSA.gov](https://groups.google.com/a/gsa.gov/g/fac-team) Google Group
+- [ ] Add to standing meeting invites (should happen automatically via addition to the Google Group membership)
+- [ ] (engineers) Add to `gsa-10x-prototyping` Organization in cloud.gov
```

Comment:
@MelissaBraxton should there be design tasks like getting added to the Figma group or anything?

Reply:
Good catch! I will add.